pyproject.toml

```diff
@@ -19,6 +19,7 @@ dependencies = [
    "tomsgeoms2d@git+https://github.com/tomsilver/toms-geoms-2d",
    "task-then-motion-planning@git+https://github.com/tomsilver/task-then-motion-planning.git",
    "relational-structs@git+https://github.com/tomsilver/relational-structs.git",
+   "pybullet-blocks@git+https://github.com/tomsilver/pybullet-blocks.git@b847436448f3e6076c9fc779c959c051c55187dc",
```

Comment:
it looks like you are pinning a specific commit -- is that intentional? just want to make sure